packages/turbo/android/src/main/java/com/reactnativeturbowebview/RNSession.kt

```diff
@@ -38,7 +36,6 @@ class RNSession(
     WebView.setWebContentsDebuggingEnabled(BuildConfig.DEBUG)
     webView.settings.setJavaScriptEnabled(true)
     webView.addJavascriptInterface(JavaScriptInterface(), "AndroidInterface")
-    setUserAgentString(webView, applicationNameForUserAgent)
```

Comment:
What's the reason for moving this into the configure WebView flow?

I'm primarily thinking that iOS doesn't support updating the user agent after the initialization of a session and wondering if it's better to keep the behavior consistent between platforms?